ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -311,11 +360,17 @@ private void readWithProgress(
       int requestId,
       ResponseBody responseBody) throws IOException {
     Reader reader = responseBody.charStream();
+    ProgressResponseBody progressResponseBody = (ProgressResponseBody)responseBody;
```

Comment:
the internal test failures are on end-to-end tests, there’s no mocking involved.